scopesim/defaults.yaml

```diff
@@ -104,23 +104,6 @@ properties :
     run_skycalc_ter_tests : True
 
 
----
-### OBSERVATION PARAMETERS
-object : observation
-name : default_observation_config
-alias : OBS
-description : observation confiugration parameters
-
-packages : []
-yamls : []
-properties : {}
-
-use_instrument : ""
-ignore_effects : []
-add_effects : []
-override_effect_values : []
-
```

Comment:
Such a set of defaults also double as documentation because it conveys to the user that they could potentially change these values. So I'd prefer to keep them if there is no drawback to having them

Reply:
IIRC, it was originally @astronomyk who said let's get rid of this. But the point now is not to play the `blame` game...

At one point of this (rebased out) I only kept the !SIM in here, but that made things fail. But at a second thought, perhaps it does make some sense to keep this as a fallback. On the other hand, if these values are _not_ set for whatever reason, and we _don't_ have these default, we would get a `KeyError` rather than an empty `list`/`dict`/`str`. The latter might produce hard-to-track-down bugs. But maybe that's not really an issue in practice. Unsure what's best...

Reply:
I reverted this now in [71c9ea2](https://github.com/AstarVienna/ScopeSim/pull/375/commits/71c9ea245257d433c80689ca3e3b422224e3e272).